src/collections/Form/index.d.ts

```diff
@@ -163,69 +145,34 @@ interface FormGroupProps {
   /** Additional classes. */
   className?: string;
 
-  /** Fields can show related choices */
+  /** Fields can show related choices. */
   grouped?: boolean;
 
-  /** Multiple fields may be inline in a row */
+  /** Multiple fields may be inline in a row. */
   inline?: boolean;
 
-  /** Fields Groups can specify their width in grid columns or automatically divide fields to be equal width */
+  /** Fields Groups can specify their width in grid columns or automatically divide fields to be equal width. */
   widths?: SemanticWIDTHS | 'equal';
 }
 
-export const FormGroup: React.ComponentClass<FormGroupProps>;
+export const FormGroup: React.StatelessComponent<FormGroupProps>;
 
-interface FormInputProps extends InputProps {
+interface FormInputProps extends FormFieldProps, InputProps {
```

Comment:
Hm, is this going to cause the duplicate property error again?  Both FormFieldProps and InputProps define a `label` prop in their interfaces, each with different types:


**FormFieldProps**
```ts
   /** Mutually exclusive with children. */
  label?: any | Object;
```

**InputProps**
```ts
  /** Optional Label to display along side the Input. */
  label?: any;
```

This may also be the case with Checkbox/Radio/Button, since they all accept a label prop as well.